### PR TITLE
Add GitHub issue template

### PR DESCRIPTION
**Pull request by kenny-evitt**
@cldwalker @rundis I took a stab at this.

Currently there's only a 'top-level' section for reporting a bug. I'm not sure what we'd want to include for any other types of issues; maybe nothing.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+## Report a Bug
+
+### *Remove this Section*
+
+Note that issues for this repo are for bugs with LightTable itself. Bugs for a specific language or other plugin should go in its plugin repository e.g. [Clojure](https://github.com/LightTable/Clojure), [JavaScript](https://github.com/LightTable/Javascript), or [Vim](https://github.com/LightTable/Vim).
+
+Questions should generally be asked on the [Light Table Google Groups group](https://groups.google.com/forum/#!forum/light-table-discussion).
```
Comment (rundis) [inline .github/ISSUE_TEMPLATE.md]: Maybe also add link to gitter ?

**Comment by kenny-evitt** [comment]
@rundis I added a link to the main Gitter room and I re-organized one sub-section to cover both the existing 'report a bug' top-level section, and a new top-level 'request a feature' section.

**Comment by rundis** [comment]
Looks good to me. Let's try it out :-)

**Comment by rundis** [comment]
🎉

**Comment by cldwalker** [comment]
@kenny-evitt Thanks for the thoroughness on this! :fireworks: Hoping this improves the quality of our bug reports

**Comment by kenny-evitt** [comment]
@rundis @cldwalker Thanks!

**Pull request merged**